### ISSUE TITLE
simreads fastq format

### ISSUE THREAD

**Issue by sagarc88** [fritzsedlazeck/SURVIVOR#223]
Hello! 

I was able to run simreads but the output is in fasta format. Is it possible to output in fastq format with simulated qualities as well? 

Thank you

**Comment by thom-99** [comment]
You may use different tools to simulated fastq reads **from a fasta referece file**

On the top of my mind I remember [badread](https://github.com/rrwick/Badread) and [NanoSim](https://github.com/bcgsc/NanoSim) which are for long-read sequencing data. 

I'm sure that you can find many other options that fit your specific requirements. 

Cheers.

**Comment by fritzsedlazeck** [comment]
Yeah sorry I wont build in that quality simulators. If you just want to convert fasta -> fastq and please use one of the existing methods out there (or perl etc. ). 

Thanks
Fritz
